examples/customsortfiltermodel/tree_cell.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	qt "github.com/mappu/miqt/qt6"
+	"strconv"
+)
+
+type TreeCell struct {
+	data map[qt.ItemDataRole]string
+}
+
+func NewTreeCell(role qt.ItemDataRole, value qt.QVariant) *TreeCell {
+	data := make(map[qt.ItemDataRole]string)
+	data[role] = value.ToString()
+	return &TreeCell{data: data}
+}
+
+func (cell *TreeCell) SetData(data qt.QVariant, role qt.ItemDataRole) {
+	cell.data[role] = data.ToString()
+}
+
+func (cell *TreeCell) Data(role qt.ItemDataRole) *qt.QVariant {
+	if role == qt.UserRole {
+		number, err := strconv.Atoi(cell.data[role])
+		if err != nil {
+			fmt.Printf("Error converting %s to int: %v\n", number, err)
```

Comment:
When starting up the example, I got these terminal messages:

```
Error converting %!s(int=0) to int: strconv.Atoi: parsing "": invalid syntax
Error converting %!s(int=0) to int: strconv.Atoi: parsing "": invalid syntax
Error converting %!s(int=0) to int: strconv.Atoi: parsing "": invalid syntax
Error converting %!s(int=0) to int: strconv.Atoi: parsing "": invalid syntax
Error converting %!s(int=0) to int: strconv.Atoi: parsing "": invalid syntax
Error converting %!s(int=0) to int: strconv.Atoi: parsing "": invalid syntax
```

They repeat when attempting to sort by the Name column,

Reply:
Ah thanks I will fix that!